android/src/main/java/com/imagepicker/ImagePickerModule.java

```diff
@@ -277,6 +275,7 @@ public void onActivityResult(int requestCode, int resultCode, Intent data) {
     }
 
     response.putString("uri", uri.toString());
+    response.putString("urlPath", realPath);
```

Comment:
Are you sure about that ? Most common usage for image is to send the data in base64 ! Are you sure you don't have a module or something to convert your path into data then send it to S3 ?